content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -8,8 +8,17 @@ weight: 10
 
 This page will show you how to get started with OpenTelemetry in Node.js.
 
-You will learn how you can instrument a simple application automatically, in
-such a way that [traces][], [metrics][] and [logs][] are emitted to the console.
+You will learn how to automatically instrument both [traces][] and [metrics][]  
+and log them to the console.
+ [Automatically](https://opentelemetry.io/docs/concepts/instrumentation/automatic/) 
+instrumenting means that the application is made observable without 
+touching the application code.
```

Comment:
Please keep the original wording just without logs. I agree that there is room for improvement but because of [this discussion](https://github.com/open-telemetry/opentelemetry.io/issues/3228) let's keep those changes in wording around automatic instrumentation minimal

Reply:
Well noted @svrnm